go.mod

```diff
@@ -7,13 +7,12 @@ require (
 	github.com/armon/go-metrics v0.3.9
 	github.com/bgentry/speakeasy v0.1.0
 	github.com/btcsuite/btcd v0.22.0-beta
-	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/coinbase/rosetta-sdk-go v0.6.10
 	github.com/confio/ics23/go v0.6.6
+	github.com/cosmos/btcutil v1.0.3-0.20210919120910-bce3713586ff
```

Comment:
Can we use an actual release?

Reply:
Hmm, it seems the tag name `v1.0.3` in go package mirror is corrupted. Even though I create the tag with the latest commit, it indicates old commit and causes an error every time. I will update PR after creating `v1.0.4` tag.